### PR TITLE
blackbox: clear stratisd vendor directory if it exists

### PR DESCRIPTION
**Pull request by bgurney-rh**

### CODE REVIEW

File: blackbox/stratis-blackbox-run.sh [hunk 1]
```diff
@@ -80,6 +80,15 @@ git clone $STRATISD_REPO
 echo "Most recent commits for ${STRATISD_N}:"
 ( cd $STRATISD_N; git status; git log --format="%h %ci :: %s" | head -20 )
 mv -v ${STRATISD_N} ${STRATISD_N}-${STRATISD_V}
+
+# If the vendor directory exists, remove it and recreate it.
```
Comment (mulkieran) [inline blackbox/stratis-blackbox-run.sh:84]: Can you just say remove it in here and the message, not recreate, since cargo will do the recreating later?

Comment (bgurney-rh) [reply blackbox/stratis-blackbox-run.sh]: Resolved.

**Comment by bgurney-rh** [comment]
Oh, I didn't know cargo would make the directory on its own.  Stand by; I want to confirm this with another test run.

(I have a working branch of the ci blackbox test where I'm testing the ability to run with an existing repository directory, which is how I found the issue that this PR fixes.)

**Comment by bgurney-rh** [comment]
Okay, it seems to work, so I removed the `mkdir` command.

**Pull request merged**